src/tcgwars/logic/impl/gen3/Emerald.groovy

```diff
@@ -459,15 +459,24 @@ public enum Emerald implements LogicCardInfo {
           resistance WATER, MINUS30
           pokeBody "Green Essence", {
             text "As long as Sceptile is in play, each of your Active Pokémon that has [G] Energy attached to it can’t be affected by any Special Conditions."
+            def greenEssence={
```

Comment:
Why define a closure that only gets used once?

Reply:
Honestly, was basing it on Dragon Majesty's Phione, which does the same. Don't see harm in it being written this way, but you do have a point.